src/components/ProjectFooter.js

```diff
@@ -23,20 +16,18 @@ const ProjectFooter = () => {
   ];
 
   return (
-      <Footer
-        size="slim"
-        primary={<FooterNav size="slim" links={footerNavItems} />}
-        secondary={
-          <Logo
-            size="slim"
-            image={<img className="usa-footer__logo-img" src="" alt="" />}
-            heading={
-              <p className="usa-footer__logo-heading">{t("agency")}</p>
-            }
-          />
-        }
-      />
+    <Footer
+      size="slim"
+      primary={<FooterNav size="slim" links={footerNavItems} />}
+      secondary={
+        <Logo
+          size="slim"
+          image={<img className="usa-footer__logo-img" src="" alt="" />}
```

Comment:
Are you going to add an image src in the future? Otherwise I don't know if you need to include this image prop.